project-koku/koku
# Fixing a few OCI bugs

## Jira Ticket

[COST-2832](https://issues.redhat.com/browse/COST-2832)

## Description

This change will fix a few things
- current_month_data flag
- Storage data SQL
- Instance type SQL

## Testing

1. Checkout Branch
2. Restart Koku
3. Ingest OCI data
4. Check you have data in the correct tables for the above

## Notes

...
